### PR TITLE
[EDU-4102] cli logs

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue: https://aziontech.atlassian.net/browse/EDU-4102

Changes: 

- Add cli logs command docs

### CODE REVIEW

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/logs/logs.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+title: Logs do Azion CLI
+description: Acompanhe os logs de suas edge functions e eventos https de suas edge applications através da Azion CLI.
+meta_tags: cli, next, javascript, astro, vue, react, edge computing
+namespace: documentation_cli_logs
+menu_namespace: cliMenuAlpha
+permalink: /documentacao/devtools/cli/logs/
+---
+
+Acompanhe os logs de suas edge functions e eventos https de suas edge applications através da Azion CLI.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/logs/logs.mdx:10]:
```suggestion
Acompanhe os logs de suas edge functions e eventos HTTP de suas edge applications através da Azion CLI.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+The `--help` option displays more information about the `azion logs cells` command.
+
+#### limit
+
+The `--limit` option defines how many logs will be displayed per request. By default, it's 100.
+
+#### pretty
+
+The `--pretty` option displays the logs in a prettified way, making it easier to read and analyze.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx:48]:
```suggestion
The `--pretty` option displays the logs in a prettified view, making it easier to read and analyze.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+#### tail
+
+The `--tail` option displays the logs continuously, similar to the `tail -f` command in Unix-like operating systems. This is useful for monitoring logs in real-time.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx:52]:
```suggestion
The `--tail` option displays the logs continuously, similar to the `tail -f` command in Unix-like operating systems. This is useful for monitoring logs in real time.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+:::note 
+This command returns the logs from the last 5 minutes.
+:::
+
+---
+
+## Edge application http logs
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx:60]: HTTP?

Comment (gabriel-azion) [reply src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx:60]:
```suggestion
## Edge application HTTP logs
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+### Optional Flags
+
+#### help
+
+The `--help` option displays more information about the `logs http` command.
+
+#### limit
+
+The `--limit` option defines how many logs will be shown per request. By default, it's 100.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx:79]:
```suggestion
The `--limit` option defines how many logs will be displayed per request. By default, it's 100.
```

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx (hunk 5 above)
Comment (hannah-sk) [reply src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx:79]: alinhar com o de cima

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+#### pretty
+
+The `--pretty` option displays the logs in a prettified way, making it easier to read and analyze.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/logs/logs.mdx:83]:
```suggestion
The `--pretty` option displays the logs in a prettified view, making it easier to read and analyze.
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/logs/logs.mdx [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+A flag `--function-id` especifica o ID da função cujos logs você deseja ver. Se não informado, os logs de todas as funções serão exibidos.
+
+#### help
+
+A opção `--help` exibe mais informações sobre o comando `azion logs cells`.
+
+#### limit
+
+A opção `--limit` define quantos logs serão exibidos por solicitação. Por padrão, são 100.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/logs/logs.mdx:44]:
```suggestion
A opção `--limit` define quantos logs serão exibidos por requisição. Por padrão, são 100.
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/logs/logs.mdx [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+### Flags Opcionais
+
+#### help
+
+A opção `--help` exibe mais informações sobre o comando `logs http`.
+
+#### limit
+
+A opção `--limit` define quantos logs serão mostrados por solicitação. Por padrão, são 100.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/logs/logs.mdx:79]:
```suggestion
A opção `--limit` define quantos logs serão exibidos por requisição. Por padrão, são 100.
```

**Review by hannah-sk — APPROVED** [review]

**Pull request merged**